### PR TITLE
Upgrade for beta5

### PR DESCRIPTION
**Pull request by agubler**
Make lots and lots of changes based on `beta5` changes.

Includes quite a few updates to tutorials specifically the deploying to production one but they could do with a proper review.

### CODE REVIEW

File: site/source/tutorials/001_static_content/index.md [hunk 1]
```diff
@@ -27,7 +27,11 @@ You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For
 
 Before we start making changes, let's start the application with the development server so that we can observe the impact of our changes. Run the following command in the application's root directory:
 
-`dojo build -w`
+`dojo build --mode dev --watch memory --serve`
+
+or using the short params:
```
Comment (dylans) [inline site/source/tutorials/001_static_content/index.md]: or using the short params: -> or using the abbreviated parameters:

File: site/source/tutorials/002_creating_an_application/index.md [hunk 2]
```diff
@@ -79,7 +79,22 @@ Our demo application includes some tests to verify that it is working as expecte
...
 
-This test is ensuring that the rendering function is returning the correct tag and that the tag has the correct content. We will return to the topic of testing in a later tutorial, but for now you can use them to check your work as you progress through this series by running the following terminal command:
+This test is ensuring that the rendering function is returning the correct tag and that the tag has the correct content. We will return to the topic of testing in a later tutorial, but for now you can use them to check your work as you progress through this series by running the following terminal commands:
+
+```bash
+dojo build --mode test
+dojo test
+```
+
+The tests can be built using a watch that means the full application doesn't need to be rebuilt to re-run the tests after every change.
```
Comment (dylans) [inline site/source/tutorials/002_creating_an_application/index.md]: The tests can be built using a watch that means the full application doesn't need to be rebuilt to re-run the tests after every change. -> The tests can be built using a watch command, meaning that the full application does not need to be rebuilt to re-run the tests after every change.

File: site/source/tutorials/1020_registries/index.md [hunk 3]
```diff
@@ -96,31 +96,27 @@ First we need to extract the `_renderBack` function from `Worker.ts` into a new
...
 {% endaside %}
 
-Now we need to add the `registry` definition for `WorkerBack.ts` to lazily load when the worker is clicked. Instead of adding a concrete widget class, we add a function that, when called, requires and returns the concrete widget. This function will not be called until the first time the application tries to use the widget label as part of the usual `render` cycle. Initially, before the widget has loaded, nothing will be rendered. Once it has loaded, any widgets that use the lazy widget will automatically re-render.
+Now we need to add the `registry` definition for `WorkerBack.ts` to lazily load when the worker is clicked. Instead of adding a concrete widget class, we add a function that, when called, dynamically imports the widget and returns the a promise that returns the widget. This function will not be called until the first time the application tries to use the widget label as part of the usual `render` cycle. Initially, before the widget has loaded, nothing will be rendered. Once it has loaded, any widgets that use the lazy widget will automatically re-render.
```
Comment (dylans) [inline site/source/tutorials/1020_registries/index.md]: returns the a promise that returns the widget ->  returns a promise that returns the widget

File: site/source/tutorials/1020_registries/index.md [hunk 4]
```diff
@@ -96,31 +96,27 @@ First we need to extract the `_renderBack` function from `Worker.ts` into a new
...
 
-{% task 'Import `load` from `@dojo/core/load` in `main.ts`' %}
+There are two ways to register a widget in a registry, the first is to define the item in the global registry as demonstrated in the `main.ts` file. This method makes the widget available to the entire application, if the widget is only needed by a single widget then the registry item can be defined using the `@registry` decorator from `@dojo/widget-core/decorators/registry`.
```
Comment (dylans) [inline site/source/tutorials/1020_registries/index.md]: demonstrated in the `main.ts` file -> demonstrated in `main.ts`

File: site/source/tutorials/1040_dojo1_dijits/index.md [hunk 5]
```diff
@@ -57,6 +57,10 @@ The `dojo`, `dijit`, and `dojo-themes` packages are the Dojo 1 dependencies we w
 
 Dojo 1 and Dijits at run-time are considered _external_ dependencies for the Dojo 2 build system.  We need to supply information to the `dojo build` CLI command to include our external dependencies as part of our build.  We provide additional configuration information to the Dojo 2 CLI via `.dojorc` in the root of our project.  `.dojorc` is formatted as a JSON file and configuration can be provided to each CLI command.
 
+{% aside 'Dojo Build Command' %}
+Currently only the older command `cli-build-webpack` supports loading external modules into Dojo 2, so for this tutorial this is the build command that is required.
```
Comment (dylans) [inline site/source/tutorials/1040_dojo1_dijits/index.md]: Should we open an issue to remind us to update this for RC1?

Comment (agubler) [reply site/source/tutorials/1040_dojo1_dijits/index.md]: There is an issue open on `cli-build-app` to support externals https://github.com/dojo/cli-build-app/issues/7

**Pull request merged**